doc/source/release-notes/Percona-Server-8.0.16-7.rst

```diff
@@ -19,33 +20,46 @@ enterprise-grade features developed by Percona.
 
 Encryption Features General Availability (GA)
 ================================================================================
-- Temporary File Encryption (Temporary File Encryption)
-- InnoDB Undo Tablespace Encryption
-- InnoDB System Tablespace Encryption (InnoDB System Tablespace Encryption)
-- :variable:`default_table_encryption`=OFF/ON (General Tablespace Encryption)
-- :variable:`table_encryption_privilege_check`=OFF/ON (Verifying the Encryption Settings)
-- InnoDB redo log encryption (for master key encryption only) (Redo Log Encryption)
-- InnoDB merge file encryption (Verifying the Encryption Setting)
-- Percona Parallel doublewrite buffer encryption (InnoDB Tablespace Encryption)
 
-.. note::
+The encryption features are described in `Data at Rest Encryption
+<https://www.percona.com/doc/percona-server/LATEST/management/data_at_rest_encryption.html>`__.
+
```

Comment:
use internal :ref: links instead of http:// this will automatically link to correct docs.